packages/stream-model-handler/src/schema-utils.ts

```diff
@@ -1,5 +1,32 @@
 
-import ajv, { AnySchema } from 'ajv/dist/2020'
+import ajv, { SchemaObject } from 'ajv/dist/2020'
+
+type JSONSchemaObjectType = {
+  type: 'object',
+  additionalProperties?: boolean
+}
+
+function traverseSchemaRecursivelyAndApplyFunction(o: SchemaObject, fn: (o: object) => void) {
+  fn(o)
+  Object.getOwnPropertyNames(o).forEach( (name) => {
+    fn.apply(this,[o[name]]);  
```

Comment:
isn't this redundant given the recursive call will cause the fn call on line 10 to run on the subobj anyway?

Reply:
Awesome catch, thanks @stbrody !